app/src/main/java/org/mozilla/reference/browser/SampleRequestInterceptor.kt

```diff
@@ -0,0 +1,17 @@
+package org.mozilla.reference.browser
+
+import android.content.Context
+import mozilla.components.browser.errorpages.ErrorPages
+import mozilla.components.browser.errorpages.ErrorType
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.request.RequestInterceptor
+
+class SampleRequestInterceptor(private val context: Context) : RequestInterceptor {
```

Comment:
nit: Let's give that class a real name :)

Reply:
Hmm.. I've realized that I'm bad with naming.

How about `NetworkRequestInterceptor`? 🤔

Reply:
Firefox for Fire TV uses `CustomContentRequestInterceptor`. Maybe just `ContentRequestInterceptor`?